packages/odyssey-react-mui/src/DataTable/DataTable.tsx

```diff
@@ -480,7 +480,7 @@ const DataTable = ({
 
   const emptyState = useCallback(() => {
     const noResultsInnerContent = noResultsPlaceholder || (
-      <DataTableEmptyState
+      <EmptyState
```

Comment:
Maybe we should rename this to `EmptyDataState` instead.

Reply:
I'm indifferent. I don't see a huge difference between the two names. Theoretically this component should be used anywhere where the data powering the UI does not exist, doesn't matter if it is a Table or Stacks.

Reply:
I'd vote to keep this as Empty State, since it is meant to be a generic empty state that can be used anywhere (not just in Data Table or Data Stack)

Reply:
I agree with Jordan, we have a generic empty state component on our roadmap so this is a good starting point for that IMO